userspace/libscap/libscap.pc.in

```diff
@@ -9,4 +9,4 @@ Version: @FALCOSECURITY_LIBS_VERSION@
 Requires: @LIBSCAP_PKGCONFIG_REQUIRES@
 Requires.private: @LIBSCAP_PKGCONFIG_REQUIRES_PRIVATE@
 Libs: -L${libdir} @LIBSCAP_LINK_LIBDIRS_FLAGS@ @LIBSCAP_LINK_LIBRARIES_FLAGS@
-Cflags: -I${includedir} -I${includedir}/libscap -I${includedir}/driver -I@UTHASH_INCLUDE@
```

Comment:
This looks like he wrong solution, to simply drop -I@UTHASH_INCLUDE@, which is correct for the non-bundle uthash case.  I think a proper solution would be to have an abstracted variable, e.g. `UTHASH_PKGCONFIG_INCLUDE` that would be set to the current value (`${UTHASH_INCLUDE}`) when `NOT USE_BUNDLED_UTHASH` or to `${CMAKE_INSTALL_INCLUDEDIR}` when `USE_BUNDLED_UTHASH` is true.

Reply:
Possibly. I'll come up with a patch though I'll rely on you to test it.

I'm also thinking of submitting a patch that just builds the Rust SDK in CI (as a way to validate the pkgconfig stuff, not the SDK itself). Is there a chance you could come up with a CI build action so that we don't need to repeat this tug-of-war? :) I fix my use case and break yours, you fix yours and break mine, GOTO 10 :)

Looks like we're using the pkgconfig files in just about the opposite ways, so if both our tests pass in CI (and @geraldcombs doesn't complain), we should cover most of the usages.

Reply:
A CI action would be something like:
```
1. install guix
2. guix pull (update to latest version)
3. guix build sysdig --with-git-url=falcosecurity-libs=https://github.com/falcosecurity/libs --with-commit=falcosecurity-libs=$COMMIT_ID
```

Reply:
I've just tried the above manually now that I can build successfully the master branch on Guix, and sysdig built and works correctly even with your changes.  I guess if the UTHASH include ever poses a problem, we can look into it again then.